.github/workflows/tests.yml

```diff
@@ -1,6 +1,11 @@
 name: neofs-sdk-go tests
 
 on:
+  push:
+    branches:
+      - master
+    paths-ignore:
+      - '*.md'
```

Comment:
```suggestion
      - '**/*.md'
```
not sure it changes anything to us now but this it how it is done below

Reply:
Fixed.